modules/persistent-storage-csi-ovirt-dynamic-provisioning.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-ovirt.adoc
+
+[id="persistent-storage-csi-ovirt-dynamic-provisioning_{context}"]
+= Dynamically provisioning oVirt CSI volumes
+
+{product-title} installs a StorageClass for each available oVirt share type.
+
+The YAML files that are created are completely decoupled from oVirt and from its Container Storage Interface (CSI) plug-in. As an application developer, you can dynamically provision ReadWriteMany (RWX) storage and deploy Pods with applications that safely consume the storage using YAML manifests. You can also provision other access modes, such as ReadWriteOnce (RWO).
+
+You can use the same Pod and persistent volume claim (PVC) definitions on-premise that you use with {product-title} on AWS, GCP, Azure, and other platforms, with the exception of the storage class reference in the PVC definition.
```

Comment:
We've been spelling out public cloud platform names on first use.

Reply:
file deleted